bin/oref0-bluetoothup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+MAC=$1
+
+if ( ifconfig wlan0 | grep -q "inet addr" ) && ( ifconfig bnep0 | grep -q "inet addr" ); then
+   sudo bt-pan client $MAC -d
+fi
+
+if ! ( hciconfig -a | grep -q "PSCAN" ) ; then
+   sudo killall bluetoothd
+   sudo /usr/local/bin/bluetoothd --experimental &
+fi
+
+if ( hciconfig -a | grep -q "DOWN" ) ; then
```

Comment:
Now that this is the same action as for PSCAN, should we make it an || check and consolidate them?